sdk/iot/inc/az_iot_hub_sas.h

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include <az_span.h>
+#include <az_result.h>
+#include <time.h>
+#include "az_iot_mqtt.h"
+#include "az_iot_hub_client.h"
+
+#ifndef _az_IOT_SAS_H
+#define _az_IOT_SAS_H
+
+#include <_az_cfg_prefix.h>
+
+typedef struct az_iot_hub_sas {
+    struct {
+        az_span audience;

```

Comment:
should this be named audience? Is this a native term for sas keys? I searched online and nothing came up.

Reply:
That's what we named it in other SDKs. It's also internal - apps shouldn't use it.

Reply:
From https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-security#security-tokens I think the correct name is "signature" so I changed to that.